src/lib/holocene/menu/menu.svelte

```diff
@@ -47,8 +48,12 @@
   class:hidden={!$open}
   aria-labelledby={id}
   tabindex={-1}
+  style={position === 'topRight' || position === 'topLeft'
+    ? `top: -${height + 16}px;`
+    : ''}
```

Comment:
Could we do this in the css declarations below

```css
.topRight,
.topLeft {
  @apply -top-4;
}
```

Reply:
Need to keep the style because it's dynamic based on the height of the menu

Reply:
Ahh right